pkg/client/plans.go

```diff
@@ -16,22 +15,13 @@ type PlansAPIService struct {
 	Cfg    Configuration
 }
 
-/*
```

Comment:
Why is this comment being removed? any specific reason?

Reply:
Removed by mistake, will adds back